mxdz/lab1-BasicIO/user.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#ifdef __XC32
+#include <xc.h>          
+#endif
+
+#include <stdint.h>       
+#include <string.h>
+#define LED_1        LATGbits.LATG6
+#define LED_2       LATDbits.LATD4
+#define LED_3        LATBbits.LATB11
+#define LED_4      LATGbits.LATG15
+
+#define BTN_1       PORTAbits.RA5
+#define BTN_2       PORTAbits.RA4
+
+#define BLINK_DELAY 10000000
+#define DEBOUNCE_DELAY 2000000
+#define MSG "test msg"
```

Comment:
It's better to define it as a constant char array to allow further GCC optimizations:
`const uint8_t MSG[] = "test msg";`